loadJsonFile.py

```diff
@@ -0,0 +1,55 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# Load file -> load 'result.json'
+with open('result.json') as f:
+    read_data = f.read()
+
+# Dict type is 'list'
+datainfo = json.loads(read_data)
+
+data = dict()
+
+# If the information is unknown, save 'unknown'
+for item in datainfo:
+    try: data['name'] = item['name']
+    except KeyError: data['name'] = 'unknown'
```

Comment:
I wonder if you want to try/except for each field here - it may leave too much error log, given not many locations has all these fields populated. Maybe we can just use a simple check?

if 'name'  in item: data['name'] = item['name'].

Also, why do we need "unknown" for all these fields? Couldn't we just check for existence of these keys in the dict when we need to consume these fields?

Reply:
For example, If you get the value for the 10 key values, if there is no value in the 3rd key, you will skip the remaining 7 key values and go to the exception handling (in python). So we have an exception handling for each key-value.

And when we go to the exception handling, I am going to modify item ['name'] = "" and item ['name'] = "unknown"

I decided to treat it as blank even when there is no value for key.